gpt/commit.js

```diff
@@ -252,6 +252,61 @@ const runGenerativeCommit = async () => {
     }
```

Comment:
The model is being reinitialized each time the recursiveAgent function is called. Consider initializing it once and passing it as an argument to the function to improve performance.